onnx/__init__.py

```diff
@@ -86,8 +87,8 @@ def _save_bytes(content: bytes, f: Union[IO[bytes], str]) -> None:
             writable.write(content)
 
 
-def _get_file_path(f: Union[IO[bytes], str]) -> Optional[str]:
-    if isinstance(f, str):
+def _get_file_path(f: Union[IO[bytes], str, Path]) -> Optional[str]:
+    if isinstance(f, str) or isinstance(f, Path):
```

Comment:
nit:
```suggestion
    if isinstance(f, (str, Path)):
```

Reply:
Fair point. I'll add this in after merging again.